api/src/test/java/org/pmiops/workbench/user/UserAdminServiceTest.java

```diff
@@ -103,4 +104,40 @@ public void testGetActiveWindow_noActiveWindow_null() {
     // does not throw
     assertThat(userAdminService.getCurrentEgressBypassWindow(null)).isNull();
   }
+
+  @Test
+  public void tesListWindows() {
+    Instant startTime1 = FakeClockConfiguration.NOW.toInstant().minus(1, ChronoUnit.DAYS);
+    Instant endTime = FakeClockConfiguration.NOW.toInstant().plus(1, ChronoUnit.DAYS);
+    Instant startTime2 = startTime1.plus(1, ChronoUnit.DAYS);
```

Comment:
this is just `NOW` - let's use that, to make it more clear.

Reply:
Done